src/dataframerow/dataframerow.jl

```diff
@@ -76,17 +76,19 @@ julia> Vector(df[1, :])
 struct DataFrameRow{D<:AbstractDataFrame,S<:AbstractIndex}
     df::D
     colindex::S
-    row::Int
+    row::Int # row number in true parent DataFrame
```

Comment:
Maybe we can find better names to distinguish these? Even before this PR, having a field called "row" in a row object is kind of weird. :-)

Reply:
Sure - maybe `parentrow` and change `df` to `parent`?

Reply:
That or just `idx`.

Reply:
I have changed `row` to `dfrow` and added some comments